### PR TITLE
WPF 886828 - Updated review corrections for Gantt migration content

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated the review corrections for Gantt UG migration content.

### CODE REVIEW

File: wpf/Gantt/migrate-from-sftreegrid.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ To facilitate the migration from [`GridTreeControl`](https://help.syncfusion.com
 ## Assembly references
 <table>
 <tr>
```
Comment (Karthickmani97) [inline wpf/Gantt/migrate-from-sftreegrid.md:15]: Migrate from SfTreegrid, alternate name ?

Comment (TamilarasanGunasekaran) [reply wpf/Gantt/migrate-from-sftreegrid.md:15]: Updated.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Comment by Karthickmani97** [comment]
@TamilarasanGunasekaran , Ask CI team to re-initiate CI for this PR.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Pull request merged**